qcodes/utils/wrappers.py

```diff
@@ -263,6 +263,7 @@ def do1d(inst_set, start, stop, num_points, delay, *inst_meas):
     """
 
     # try to flush VISA buffers at the beginning of a measurement
```

Comment:
It is slightly confusing that `interrupted` is set on the line between the buffer flushing and the comment about buffer flushing.